CONTRIBUTING.md

```diff
@@ -51,5 +51,5 @@ Based on [PHPStan](https://github.com/phpstan/phpstan).
 ## Running benchmarks
 Benchmarks are run via [PHPBench](https://github.com/phpbench/phpbench).
 ```sh
-./vendor/bin/phpbench run .
+./vendor/bin/phpbench run benchmarks
```

Comment:
Have you tried `vendor/bin/phpbench run`?

Reply:
I just tried and it fails for me for some reason indeed. While `./vendor/bin/phpbench run benchmarks` works.

Reply:
Ah, that works, too. Should I open another PR?